src/pages/jeongaYeon/Login/Login.scss

```
@@ -0,0 +1,64 @@
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+.login {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  align-items: center;
+  width: 400px;
+  margin: 100px auto;
+  padding: 60px;
+  border: 1px solid rgba(128, 128, 128, 0.5);
+  border-radius: 2px;
+
+  .loginTitle {
+    margin-bottom: 50px;
+    font-family: 'Lobster', cursive;
+    font-size: 30px;
+  }
+
+  .loginIdWrapper,
+  .loginPasswordWrapper {
+    width: 100%;
+  }
+
+  .loginId,
+  .loginPassword {
+    width: 100%;
+    height: 45px;
+    margin-bottom: 3px;
+    padding-left: 10px;
+    border: 1px solid rgba(128, 128, 128, 0.5);
+    border-radius: 3px;
+    background-color: #fafafa;
+  }
+
+  .loginPassword {
+    margin-top: 10px;
+  }
+
+  .loginButton {
+    width: 100%;
+    height: 35px;
+    margin-top: 20px;
+    margin-bottom: 180px;
```

```suggestion
    margin: 20px 0 180px ;
```
가능하다면 코드를 짧게 줄이는 것이 좋습니다!
위의 두 스타일은 이렇게 합칠 수 있겠습니다.